src/EnergyPlus/WaterCoils.cc

```diff
@@ -5136,87 +5135,66 @@ void ReportWaterCoil(EnergyPlusData &state, int const CoilNum)
     // na
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    Real64 RhoWater;
-    Real64 Tavg;
-    Real64 SpecHumOut;
-    Real64 SpecHumIn;
-    Real64 ReportingConstant;
 
-    if (state.dataWaterCoils->WaterCoil(CoilNum).reportCoilFinalSizes) {
+    auto &waterCoil = state.dataWaterCoils->WaterCoil(CoilNum);
+
+    if (waterCoil.reportCoilFinalSizes) {
```

Comment:
defined ShortCut for water coils as a good practice

Reply:
Thanks @Nigusse 👍